application/settings.gradle.kts

```diff
@@ -58,7 +58,7 @@ dependencyResolutionManagement {
             addPlugin("tools.forma.demo:dependencies", "0.0.1")
             addPlugin(
                 "com.google.devtools.ksp",
-                "1.8.10-1.0.9",
+                "$embeddedKotlinVersion-1.0.9",
```

Comment:
I don't think it would work. For example, after updating to gradle-8.2 embedded kotlin version would be `1.8.20`, but there is NO ksp version `1.8.20-1.0.9`: https://github.com/google/ksp/releases